compiler/quilt/tools/store.py

```diff
@@ -50,7 +50,15 @@ def __init__(self, location=None):
 
         version = self._read_format_version()
 
-        if version not in (None, self.VERSION):
+        if version == '1.2':
```

Comment:
neat!  quick q: what happens when we have datastores that need two versions worth of upgrades?